### DISCUSSION TITLE
ScreenToGif 2.38

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1185] [Announcements]
# What's new?

• Added more keyboard shortcuts for frame navigation in the timeline (Ctrl and/or Alt while pressing left/right arrows).
• Updated the Italian and French localizations.  

## Fixed:

♦ The fade transition "to color" was not working properly for high DPI projects.  

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

I continue working on the next big release (v3), so expect less updates for v2. 
It will be a complete rewrite of the app, with nice new features for capturing, editing, and exporting content.
The target for the next version is around the end of the year, maybe sooner. ;)

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.38'>ScreenToGif 2.38</a>.</em>

**Comment by Andrzejone** [comment]
Still can't manage to get ride of infinity loading bug when waiting for editor to load after capture.

**Reply by NickeManarin** [reply]
Did you try to move your cache folder to another drive?
Also, have you tried to set the cache folder to the AV whitelist?
